lib/binaries.sh

```diff
@@ -90,3 +90,17 @@ install_npm() {
     fi
   fi
 }
+
+install_heroku_node() {
+  local dir="$1"
+
+  echo "Installing Heroku custom node binary..."
+
+  mv $dir/node $dir/node-original
+
+  touch $dir/node
+  echo "#!/usr/bin/env bash" > $dir/node
+  echo ".heroku/node/bin/node-original \$NODE_OPTS \$@" >> $dir/node
```

Comment:
We get a lot of issues claiming that there must be a problem with "Heroku's version of node", and it's great to be able to tell them that it is exactly the same binary that's downloaded from nodejs.org, so I would be hesitant to alias it like this.